diff -aurN powersave-0.13.5.orig/config_files/sleep powersave-0.13.5/config_files/sleep
--- powersave-0.13.5.orig/config_files/sleep	2006-08-14 12:57:14.000000000 +0200
+++ powersave-0.13.5/config_files/sleep	2006-08-19 19:10:56.000000000 +0200
@@ -16,7 +16,7 @@
 
 ## Path:        	System/Powermanagement/Powersave/Sleep_Modes
 ### Type:        	string
-## Default:     	"usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 ipw2200 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
+## Default:     	"usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
 ## ServiceRestart: 	
 #
 # These modules will be unloaded before entering suspend to disk
@@ -26,7 +26,7 @@
 
 ## Path:        	System/Powermanagement/Powersave/Sleep_Modes
 ### Type:        	string
-## Default:     	"usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 ipw2200 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
+## Default:     	"usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
 ## ServiceRestart: 	
 #
 # These modules will be unloaded before entering the corresponding
@@ -36,7 +36,7 @@
 
 ## Path:        	System/Powermanagement/Powersave/Sleep_Modes
 ### Type:        	string
-## Default:     	"usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 ipw2200 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
+## Default:     	"usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
 ## ServiceRestart: 	
 #
 # These modules will be unloaded before entering the corresponding
diff -aurN powersave-0.13.5.orig/scripts/sleep_helper_functions powersave-0.13.5/scripts/sleep_helper_functions
--- powersave-0.13.5.orig/scripts/sleep_helper_functions	2006-08-14 12:57:15.000000000 +0200
+++ powersave-0.13.5/scripts/sleep_helper_functions	2006-08-19 19:11:23.000000000 +0200
@@ -252,9 +252,9 @@
 # function to set the variables according to what
 # we are going to do.
 # internally, use only in sleep_helper_functions
-DEFAULT_S2D_UNLOAD="usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 ipw2200 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
-DEFAULT_S2R_UNLOAD="usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 ipw2200 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
-DEFAULT_STB_UNLOAD="usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 ipw2200 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
+DEFAULT_S2D_UNLOAD="usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
+DEFAULT_S2R_UNLOAD="usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
+DEFAULT_STB_UNLOAD="usb_storage sbp2 ohci_hcd uhci_hcd stir4200 ohci1394 rt2500 prism54 ath_pci r8169 lt_modem Intel536 Intel537 ndiswrapper"
 DEFAULT_S2D_RESTART="slmodemd irda upsd apcupsd"
 DEFAULT_S2R_RESTART="slmodemd irda upsd apcupsd"
 DEFAULT_STB_RESTART="slmodemd irda upsd apcupsd"
